src/css/styles.scss

```diff
@@ -6,16 +6,17 @@ $title-font: "Sulphur Point", sans-serif;
 $white-in-cards: hsla(357, 15%, 95%, 1);
 $black-shadow-shadow: hsla(360, 5%, 0%, 1);
 $main-blue: hsla(209, 24%, 47%, 1);
-$seafoam-h2-hover: hsla(148, 38%, 65%, 1);
```

Comment:
I think that this might break some of my code? What color would we like to move to if this is the case?

Reply:
I had to make the buttons darker with our $dark-green value. So we need to add hover state styling back in to reflect that.